### PR TITLE
Add bitarray recipe

### PR DESCRIPTION
**Pull request by KGB99**
The tests for the bitarray still do not run. I tried to write one that runs using pytest, however i get a runtime error.

### CODE REVIEW

File: recipes/recipes_emscripten/bitarray/recipe.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+tests:
+- script: pytester
+  files:
+    recipe:
+    - run_test.py
+  requirements:
+    build:
+    - pytester
+    - bitarray
```
Comment (IsabelParedes) [inline recipes/recipes_emscripten/bitarray/recipe.yaml:38]: `bitarray` shouldn't need to be included again here

File: recipes/recipes_emscripten/bitarray/recipe.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    - python
+    - pip
+  run:
+    - python
+
+tests:
+- script: pytester
+  files:
+    recipe:
+    - run_test.py
```
Comment (IsabelParedes) [inline recipes/recipes_emscripten/bitarray/recipe.yaml:34]: Could you rename this file to `test_bitarray.py` for consistency?

File: recipes/recipes_emscripten/bitarray/run_test.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import bitarray
```
Comment (IsabelParedes) [inline recipes/recipes_emscripten/bitarray/run_test.py:1]: Could you move the import inside the function?

File: recipes/recipes_emscripten/bitarray/run_test.py [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+import pytest
```
Comment (IsabelParedes) [inline recipes/recipes_emscripten/bitarray/run_test.py:2]: `pytest` shouldn't be needed here

**Review by IsabelParedes — CHANGES_REQUESTED** [review]

**Comment by DerThorsten** [comment]
I believe this error of the missing file  '/lib/python3.11/site-packages/bitarray/test_150.pickle' might be due to the filtering in empack. We are currently fixing this with https://github.com/emscripten-forge/empack/pull/98

**Review by DerThorsten — APPROVED** [review]

**Review by IsabelParedes — APPROVED** [review]

**Pull request merged**